src/components/Stories/TrendingStories.js

```diff
@@ -1,14 +1,60 @@
 import React, { Component } from 'react';
 import { Card, CardBody, CardHeader } from 'reactstrap';
+import { Media } from 'reactstrap';
+import _ from 'lodash';
+import '../../styles/TrendingStories.css';
+import { mockData } from '../../mockData/storyData';
 
 class TrendingStories extends Component {
+  constructor(props) {
+    super(props);
+    this.getStoryDetails = this.getStoryDetails.bind(this);
+    this.handleUpVoteClick = this.handleUpVoteClick.bind(this);
+    this.handleDownVoteClick = this.handleDownVoteClick.bind(this);
+  }
+
+  handleUpVoteClick(data) {
+    data.upvotes++;
+    //api call to change data
+  }
+
+  handleDownVoteClick(data) {
```

Comment:
We should probably have 2 properties: `upvotes` and `downvotes` that store how many upvotes and downvotes there are, to calculate score, we can just do `score = upvotes - downvotes` that way, we can do other kinds of math with it as well (for example, upvotes/downvotes ratio)